MetroidPrimeInterface.py

```diff
@@ -484,3 +591,35 @@ def sync_artifact_layers(self) -> bool:
                         self.set_layer_active(
                             ARTIFACT_TEMPLE_ROOM_INDEX, layer_id, item.current_amount > 0)
                         changed = True
+
+    def reset_relay_tracker_cache(self):
+        self.relay_trackers = None
+
+    def update_relay_tracker_cache(self):
+        if self.relay_trackers is None:
+            self.relay_trackers = {}
+            world_state_array = struct.unpack(">I", self.dolphin_client.read_pointer(GAMES[self.current_game]["game_state_pointer"], 0x94, struct.calcsize(">I")))[0]
+            world_states = [world_state_array + i * 0x18 for i in range(7)]

```

Comment:
Would you be open to naming the offsets as a variable that describes what it is or what its intent is? Feel free to push back if this onerous

Reply:
I'll name them